adapters/algorix/algorix.go

```diff
@@ -0,0 +1,170 @@
+package algorix
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"net/http"
+	"text/template"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type AlgoriXAdapter struct {
```

Comment:
Please use the name `adapter`. There is no need to repeat the package name and no need to export with a uppercase letter.